bigchaindb/util.py

```diff
@@ -162,8 +165,52 @@ def create_tx(current_owner, new_owner, tx_input, operation, payload=None):
 
     Raises:
         TypeError: if the optional ``payload`` argument is not a ``dict``.
-    """
 
+    Reference:
+        {
```

Comment:
I added it to the docstring as a reference while implementing the new transaction structure. It can maybe be removed